flutter/realm_flutter/linux/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+# The Flutter tooling requires that developers have CMake 3.10 or later
+# installed. You should not increase this version, as doing so will cause
+# the plugin to fail to compile for some customers of the plugin.
+cmake_minimum_required(VERSION 3.10)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.19)
```

Reply:
Why is this needed for LInux? I ask because Google warns us to keep it as low as possible, and they use 3.10 when we generate a new plugin.